cmd/backend.go

```diff
@@ -1,18 +1,124 @@
 package main
 
 import (
+	"flag"
+	"fmt"
 	"github.com/grafana/grafana-build/pipelines"
 	"github.com/urfave/cli/v2"
+	"strings"
 )
 
-var TestBackendUnit = &cli.Command{
-	Name:   "test",
-	Action: PipelineAction(pipelines.GrafanaBackendTests),
+type ChoiceFlag struct {
```

Comment:
There's enough content with ChoiceFlag that it should probably be in its own file (`choice_flag.go`).